src/assets/icons/close-white.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="none" viewBox="0 0 30 30">
```

Comment:
I would rename this icon to `xmark-white`, as we already have one with this name

Reply:
mmm... I was not aware of this icon. We should make them the same!

![image](https://user-images.githubusercontent.com/15896177/158993388-062da2dc-39a7-450a-84bc-3a83940bf030.png)

Which one you like the most?

Reply:
Done as agreed. Removed the `close-white` icon at the left and added a white version of the existing `xmark` icon.

Reply:
But instead of adding the same versión why you don't use the black one and override the fill property of the path?